src/main/java/io/teak/sdk/core/LaunchDataSource.java

```diff
@@ -154,10 +170,6 @@ public static LaunchDataSource sourceFromIntent(@NonNull final Intent intent) {
                     } catch (Exception e) {
                         Teak.log.exception(e);
                     }
-                } catch (SSLProtocolException ssl_e) {
-                    // Ignored, Sentry issue 'TEAK-SDK-Z'
-                } catch (SSLException ssl_e) {
```

Comment:
Just removing these catches is fine for now, but long term we might want to consider doing an error log (not an exception log) on ignored exceptions, just so we keep seeing them when reviewing logs, so we don't get this kind of "try to guess what happened based on a black hole" problem again.